src/map/lua/luautils.cpp

```diff
@@ -243,6 +244,8 @@ namespace luautils
         // Handle settings
         contentRestrictionEnabled = GetSettingsVariable("RESTRICT_CONTENT") != 0;
 
+        moduleutils::LoadLuaModules();
```

Comment:
Moving the loading here makes it so we can no longer add overrides for zone objects as it happens before zone loading now, Unless I am missing how to do it properly.